src/components/SearchIcon/SearchIcon.tsx

```diff
@@ -0,0 +1,10 @@
+const SearchIcon = () => {
+    return (
+        <svg id="search-svg" viewBox="0 0 50 50" fill="#FF5555" xmlns="http://www.w3.org/2000/svg">
```

Comment:
fill="currenColor" позволяет задавать цвет через стили

```suggestion
        <svg id="search-svg" viewBox="0 0 50 50" fill="currenColor" xmlns="http://www.w3.org/2000/svg">
```